allennlp/nn/activations.py

```diff
@@ -100,3 +68,21 @@ def _get_name(self):
     "tanhshrink": (torch.nn.Tanhshrink, None),
     "selu": (torch.nn.SELU, None),
 }
+
+
+@Activation.register("linear")
+class LinearActivation(Activation):
```

Comment:
Is there a reason we can't call this "Linear" like before? Worried this change could be breaking.

Reply:
Calling this "Linear" conflicts with torch's `Linear` layer. I don't think this is a breaking change, because no one was using this class directly before, so the only affect of this change is how this module is displayed like when calling `print(model)`, for example.